packages/oasis-actions/src/strategies/aave/close.ts

```diff
@@ -110,19 +242,23 @@ async function buildOperation(
     .integerValue(BigNumber.ROUND_DOWN)
 
   const closeArgs = {
-    lockedCollateralAmountInWei: args.collateralAmountLockedInProtocolInWei,
+    lockedCollateralAmountInWei: dependencies.shouldCloseToCollateral
+      ? swapData.fromTokenAmount
+      : args.collateralAmountLockedInProtocolInWei,
     flashloanAmount: amountToFlashloanInWei,
-    fee: DEFAULT_FEE,
+    fee: dependencies.shouldCloseToCollateral ? 0 : DEFAULT_FEE, //TODO - fee should be passed
```

Comment:
Why is it 0 ?

Reply:
Because for `closeToCollateral` We ain't going to collect fee. I think it's written in all the comments that I left.

Reply:
Do we need the TODO?